modules/installation-dns-user-infra.adoc

```diff
@@ -27,26 +32,127 @@ name and `<base_domain>` is the cluster base domain that you specify in the
 
 .2+a|Kubernetes API
 |`api.<cluster_name>.<base_domain>.`
-|This DNS A/AAAA or CNAME record must point to the load balancer
-for the control plane machines. This record must be resolvable by both clients
+|Add a DNS A/AAAA or CNAME record, and a DNS PTR record, to identify the load balancer
+for the control plane machines. These records must be resolvable by both clients
 external to the cluster and from all the nodes within the cluster.
 
 |`api-int.<cluster_name>.<base_domain>.`
-|This DNS A/AAAA or CNAME record must point to the load balancer
-for the control plane machines. This record must be resolvable from all the
+|Add a DNS A/AAAA or CNAME record, and a DNS PTR record, to identify the load balancer
+for the control plane machines. These records must be resolvable from all the
 nodes within the cluster.
 [IMPORTANT]
 ====
 The API server must be able to resolve the worker nodes by the host names
-that are recorded in Kubernetes. If it cannot resolve the node names, proxied
-API calls can fail, and you cannot retrieve logs from pods.
+that are recorded in Kubernetes. If the API server cannot resolve the node
+names, then proxied API calls can fail, and you cannot retrieve logs from pods.
 ====
 
 |Routes
 |`*.apps.<cluster_name>.<base_domain>.`
-|A wildcard DNS A/AAAA or CNAME record that points to the load balancer that targets the
+|Add a wildcard DNS A/AAAA or CNAME record that refers to the load balancer that targets the
 machines that run the Ingress router pods, which are the worker nodes by
-default. This record must be resolvable by both clients external to the cluster
+default. These records must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 
+|Bootstrap
+|`bootstrap.<cluster_name>.<base_domain>.`
+|Add a DNS A/AAAA or CNAME record, and a DNS PTR record, to identify the bootstrap
+machine. These records must be resolvable by the nodes within the cluster.
+
+|Master hosts
+|`<master><n>.<cluster_name>.<base_domain>.`
+|Add DNS A/AAAA or CNAME records and DNS PTR records to identify each machine
+for the master nodes. These records must be resolvable by the nodes within the cluster.
+
+|Worker hosts
+|`<worker><n>.<cluster_name>.<base_domain>.`
+|Add DNS A/AAAA or CNAME records and DNS PTR records to identify each machine
+for the worker nodes. These records must be resolvable by the nodes within the cluster.
+
 |===
+
+[TIP]
+====
+You can use the `nslookup <hostname>` command to verify name resolution. You can use the
+`dig -x <ip_address>` command to verify reverse name resolution for the PTR records.
+====
+
+The following content shows sample A records in BIND format for name resolution. The
+purpose of the sample is to show the records that are needed. The sample is not meant
+to provide advice for choosing one name resolution service over another.
+
+.Sample DNS zone database
+[%collapsible]
+====
```

Comment:
Does this formatting also work with "----" (codeblocks) instead of "====" (note blocks)?

Reply:
The doc guidelines happened to show ==== under the Collapsible content heading, so I went with it.

Reply:
Works for me. :)